lib/public/components/Filters/common/filters/TextTokensFilterModel.js

```diff
@@ -11,21 +11,21 @@
  * or submit itself to any jurisdiction.
  */
 import { Observable } from '/js/src/index.js';
+import { FilterModel } from '../FilterModel.js';
 
 /**
- * Model for text input
+ * Model which accept string input and treats it as sequence of tokens,
+ * which are validated and processed with regards of given configuration. @see TextTokensFilterModel#constructor
  */
-export class TextTokensFilterModel extends Observable {
+export class TextTokensFilterModel extends FilterModel {
     /**
      * Constructor
-     * @param {RegExp} [inputValidationPattern] regex to validate input
      * @param {RegExp|string} [splitPattern = null] regex for splitting input into tokens,
-     * it should describe delimiter between tokens. It might be single comma e.g. ',' or RegEx like /[ ,;]+/
+     * it should describe delimiter between tokens. It might be single comma e.g. ',' or RegEx like /[ ,]+/
      * By defauly it's null so input is not splitted
      */
-    constructor(inputValidationPattern, splitPattern = null) {
+    constructor(splitPattern = null) {
```

Comment:
Here I would not even put the parameter. For now you'll only split by comma, simply put this as constant somewhere and if we need to change it at some point then we'll add the parameter.
I would rather use split `,` then trim items, it's more flexible. And again, if at some point we need more complex stuff, we can improve at that time (if everything is properly factorized it should not be hard)